### PR TITLE
Translation of README.md to EN

### PR DESCRIPTION
**Pull request by theleteron**
**Changes**
- Added english version of README as `README.md`, czech version is now under `README.cs.md`
- Added `.vscode/` to `.gitignore`

Someone with better language skills than me should check this

### CODE REVIEW

**Review by janch32 — DISMISSED** [review]

**Review by Misha12 — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -18,51 +19,57 @@ sudo apt-get update;
...
+### [Linux (RHEL)](https://docs.microsoft.com/en-us/dotnet/core/install/linux-rhel)
 
 ```sh
 sudo dnf install dotnet-sdk-5.0
 ```
 
-### MacOS
+### [MacOS](https://docs.microsoft.com/cs-cz/dotnet/core/install/macos)
 
 https://docs.microsoft.com/cs-cz/dotnet/core/install/macos
```
Comment (Adrijaned) [inline README.md:30]:
```suggestion
https://docs.microsoft.com/en-US/dotnet/core/install/macos
```

File: README.md [hunk 2]
```diff
@@ -18,51 +19,57 @@ sudo apt-get update;
...
-- Cokoliv přidáte do configu, takto dejte do `appsettings.json`, aby ostatní věděli, co v tom vlastně je.
-- Všechno přidávejte formou PR. **NIKDO** nebude pushovat přímo do `master` branche.
-- Pokud nevíte. Ptejte se.
-- Dívejte se do konzole (to znamená stdout a stderr). Provádí se tam logování.
-- V projektu je využit dependency injection kontejner. Vyžaduje to knihovna Discord.NET.
-- Appka je napsaná, že by se mělo vše načítat dynamicky, tudíž, když budete chtít něco přidat, tak bude stačit udělat tyto jednoduché kroky:
-  - Vytvořte novou třídu v `Modules/`.
-  - Poděďte z bázové třídy `ModuleBase`.
-  - Užijte si zábavu. Pokud nevíte, tak se koukněte jinam.
+## What you need to know?
```
Comment (Adrijaned) [inline README.md:65]:
```suggestion
## Needed knowledge
```

File: README.md [hunk 3]
```diff
@@ -18,51 +19,57 @@ sudo apt-get update;
...
+
+- Anything you add to config also add to `appsettings.json` so that others know what is inside it.
+- Use PRs (Pull Requests) to add features or make changes. **NO ONE** can push directly to the `master` branch.
+- If you are not sure or don't know how to do something **don't feel shy to ask others** for help.
+- Check console (stdout, stderr) for logs.
+- This project uses a dependency injection container. It's required by the Discord.NET library.
+- If you want to add something just follow these steps (everything should load dynamically):
+  - Create a new class `Modules/`.
```
Comment (Adrijaned) [inline README.md:73]:
```suggestion
  - Create a new class in `Modules/`.
```

**Review by Adrijaned — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -18,51 +19,57 @@ sudo apt-get update;
...
-- Dívejte se do konzole (to znamená stdout a stderr). Provádí se tam logování.
-- V projektu je využit dependency injection kontejner. Vyžaduje to knihovna Discord.NET.
-- Appka je napsaná, že by se mělo vše načítat dynamicky, tudíž, když budete chtít něco přidat, tak bude stačit udělat tyto jednoduché kroky:
-  - Vytvořte novou třídu v `Modules/`.
-  - Poděďte z bázové třídy `ModuleBase`.
-  - Užijte si zábavu. Pokud nevíte, tak se koukněte jinam.
+## What you need to know?
+
+- Anything you add to config also add to `appsettings.json` so that others know what is inside it.
+- Use PRs (Pull Requests) to add features or make changes. **NO ONE** can push directly to the `master` branch.
```
Comment (alpatron) [inline README.md:68]: There is strictly no need for a capitalised "Pull Requests" (and Chicago style would recommend against it), but there is nothing inherently wrong with this style. Just make sure to keep consistency in the future.

File: README.md [hunk 5]
```diff
@@ -18,51 +19,57 @@ sudo apt-get update;
...
+- If you are not sure or don't know how to do something **don't feel shy to ask others** for help.
+- Check console (stdout, stderr) for logs.
```
Comment (alpatron) [inline README.md:70]: Check **the** console (stdout, stderr) for **any** logs.

**Review by alpatron — COMMENTED** [review]
Quick glance. Some polish can be applied.

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (janch32) [reply README.md:65]: No

**Review by janch32 — APPROVED** [review]

**Review by Misha12 — APPROVED** [review]

**Pull request merged**